package.json

```diff
@@ -28,10 +28,11 @@
     "lint:fix": "npm run lint -- --fix",
     "formatlib": "find warplib/ -iname *.cairo -exec cairo-format -i {} +",
     "test": "yarn warplib && tests/behaviour/setup.sh && npx mocha \"tests/**/*.test.ts\" --extension ts --require ts-node/register --parallel --exit",
```

Comment:
Imo we should also remove `yarn warplib && tests/behaviour/setup.sh`. We should check if it does not break the CI though

Reply:
I see. `setup.sh` is the script to download semantic tests. Let's leave it as it is then